src/SwipeableDrawer/SwipeableDrawer.js

```diff
@@ -188,7 +184,7 @@ class SwipeableDrawer extends React.Component {
         : event.touches[0].clientY;
 
     // We don't know yet.
-    if (this.isSwiping === undefined) {
+    if (this.isSwiping == null) {
```

Comment:
`false == null` -> false
`true == null` -> false
`undefined == null` true
`null == null` true, obviously

Reply:
Why not using `if (this.isSwiping === null) {` to clear any possible confusion?

Reply:
Because `undefined === null` is false. Doesn't make a difference here, but checking for null-ity with `==` works pretty well in Javascript.